types/sitemap-generator/tsconfig.json

```
@@ -0,0 +1,29 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictFunctionTypes": true,
+        "strictNullChecks": true,
+        "baseUrl": "../",
+        "typeRoots": [
+            "../"
+        ],
+        "types": [],
+        "paths": {
+            "@simplecrawler/*": [
+                "simplecrawler__*"
+            ]
+        },
```

```suggestion
        "types": [],
```
why do you need this? It's seems to be registered without a namespace, as `simplecrawler`?